docs/openapi.yml

```diff
@@ -1051,3 +1051,32 @@ components:
         sources:
           items:
             $ref: '#/components/schemas/source'
+    tag:
```

Comment:
The `tag` object here only defines the component, you'll also need to define the API endpoint to list tags using this object. For a great example, see the definition of [`GET /sources`](https://github.com/MarquezProject/marquez/blob/master/docs/openapi.yml#L102)